### ISSUE TITLE
Adding a Windows node fails on latest Azure WindowsServer 2019-Datacenter-with-Containers images

### ISSUE THREAD

**Issue by bashofmann** [rancher/rancher#30260]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
* Create a custom windows cluster
* Create a VM on Azure with the latest WindowsServer 2019-Datacenter-with-Containers image
* Run the join agent command on the Windows VM

**Result:**
* The node should be added to the cluster.

Error message:

```
INFO: Detecting running permission ...
INFO: Detecting host CPU reservation ...
INFO: Detecting host RAM reservation ...
INFO: Detecting host DISK reservation ...
INFO: Detecting host Docker name pipe existing ...
INFO: Detecting host Docker release version ...
FATA: Only support with Docker EE
```

**Other details that may be helpful:**
The image comes with a docker container runtime pre-installed.  The version is now reported as `Mirantis Container Runtime` instead of `Docker Enterprise`:

```
 docker version
Client: Mirantis Container Runtime
 Version:           19.03.13
 API version:       1.40
 Go version:        go1.13.15
 Git commit:        0c38b2d
 Built:             11/12/2020 21:33:08
 OS/Arch:           windows/amd64
 Experimental:      false

Server: Mirantis Container Runtime
 Engine:
  Version:          19.03.13
  API version:      1.40 (minimum version 1.24)
  Go version:       go1.13.15
  Git commit:       5b5efb2d49
  Built:            11/12/2020 21:31:44
  OS/Arch:          windows/amd64
  Experimental:     false
```

Because of that, this check in the Windows bootstrap script does not work anymore: https://github.com/rancher/rancher/blob/8a14c935aa3b573c437ea04d89843332228dc471/package/windows/bootstrap.ps1#L173-L177

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): 2.5.2
- Installation option (single install/HA): HA

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Custom with Windows support
- Machine type (cloud/VM/metal) and specifications (CPU/memory): Azure, latest WindowsServer 2019-Datacenter-with-Containers image
- Kubernetes version (use `kubectl version`): v1.18.12
- Docker version (use `docker version`): Mirantis Container Runtime 19.03.13

**Comment by bashofmann** [comment]
Just noticed, that there is already a fix for this in the 2.5 branch, which is not (yet?) in master: https://github.com/rancher/rancher/commit/670f21c6d50e5dba80f9576cd7c6361fa6c2eda5#diff-699f8fb88e10460b51809d18260b3cf61f3d95d48dcea54cf9ebddc70bd5e638.

**Comment by superseb** [comment]
Duplicate of https://github.com/rancher/rancher/issues/30148
